tests/phpunit/tests/cron.php

```diff
@@ -1022,6 +1025,48 @@ public function test_schedule_short_circuit_with_false_returns_error_when_wp_err
 		$this->assertSame( 'pre_reschedule_event_false', $rescheduled_event->get_error_code() );
 	}
 
+	/**
+	 * @ticket 57271
+	 *
+	 * @dataProvider data_wp_reschedule_event_args
```

Comment:
Data providers used by a single test should be named by replacing the test prefix with data. [See docs](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#repetitive-tests).

```suggestion
	 * @dataProvider data_wp_reschedule_event_works_with_args
```